src/omero_zarr/masks.py

```diff
@@ -35,7 +35,7 @@ def image_masks_to_zarr(image: omero.gateway.Image, args: argparse.Namespace) ->
 
     conn = image._conn
     roi_service = conn.getRoiService()
-    result = roi_service.findByImage(image.id, None)
+    result = roi_service.findByImage(image.id, None, {"omero.group": "-1"})
```

Comment:
This probably fails under group sudo but I expect that nobody will notice.

Reply:
So under group sudo one would need to get the group of the image and use that?

Reply:
If I recall correctly, one's very much locked into the group, so it's more a case of really hoping that the image is in that group. The server could be fixed to better tolerate all-groups context for group sudo so at least there is no exception if the groups do match but I don't think it's a trivial change.

Reply:
(For other CLI all-groups stuff I was thinking the client could simply not attempt to set that context at all if in group sudo.)

Reply:
Ok. But there haven't been any changes client-side to implement that, right? If so, I assume this will need to be picked up with the rest.

Reply:
Right. Fine with me to have this as-is while it awaits that.

Reply:
If there's an existing ticket for that issue, please feel free to register this PR with it.